website/content/docs/agent-and-proxy/autoauth/methods/azure.mdx

```diff
@@ -16,7 +16,17 @@ on the value of the `resource` parameter.
 
 - `role` `(string: required)` - The role to authenticate against on Vault
 
-- `resource` `(string: required)` - The resource name to use when getting instance information
+- `authenticate_from_environment` `(boolean: false)` - If set to true, will attempt to make an Azure
+  identity object through Azure specific environment variables, in priority order and as specified [in
+  Azure documentation][environment-variables]. Will work seamlessly with Azure
+  [Workload Identity][workload-identity] and others. If set to true, `object_id`, and
+  `client_id` will be ignored in favour of this authentication approach.
```

Comment:
❓ Do you think we should add a `~> Note:`  for this part:

> If set to true, `object_id`, and `client_id` will be ignored in favour of this authentication approach.

Reply:
Good idea! I didn't think that was common for config value lists like this, but it seems it's done in other places